example-get-started-experiments/code/README.md

```diff
@@ -108,6 +108,17 @@ This tag also contains a GitHub Actions workflow that reruns the pipeline if any
 [CML](https://cml.dev/) is used in this workflow to provision a cloud-based GPU 
 machine as well as report model performance results in Pull Requests.
 
+## Deploying the model
+
+Check out the [PR](https://github.com/iterative/example-get-started-experiments/pulls)
+that adds this model to 
+[Iterative Studio Model Registry](https://dvc.org/doc/studio/user-guide/model-registry/what-is-a-model-registry).
+You can [trigger CI/CD](https://dvc.org/doc/studio/user-guide/model-registry/use-models)
```

Comment:
Can we put a link to the GH action file here?

Reply:
```suggestion
You can [trigger CI/CD](https://dvc.org/doc/studio/user-guide/model-registry/use-models#deploying-and-publishing-models-in-cicd)
```

Reply:
Putting a link to the section that explains CI/CD (forgot to add reference to exact part of the page there).